setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 name = django-push-notifications
-version = 1.5.0
-description = Send push notifications to mobile devices through GCM, APNS or WNS in Django
+version = 1.6.0
+description = Send push notifications to mobile devices through GCM, APNS, WNS or WebPush in Django
```

Comment:
This should be updated if notifications can also be sent to browsers.

Reply:
Yes of course

Reply:
Done [ca70992ea2f0e2b13d460c4adba5690bd4c2bc1e]